main/com.spotify/manifest.tt

```diff
@@ -283,6 +283,16 @@ class @com.spotify
   #[confirm=enum auto]
   #[minimal_projection=[]];
 
+  action create_playlist(in req playlistName : String)
```

Comment:
we should probably choose `name` as the parameter name (shorter and more generic, makes it easier for the neural models we adopted recently), and we should write a whole bunch of canonical forms around it.